<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelUTSetupSurfaceTracing</class>
 <widget class="QWidget" name="PanelUTSetupSurfaceTracing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>86</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="surfaceTracing">
     <property name="text">
      <string>表面跟踪</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="surfaceContainer" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="1" column="3">
       <widget class="QLabel" name="latest">
        <property name="text">
         <string>最晚触发时间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="earliest">
        <property name="text">
         <string>首次触发时间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="surfaceEarliest">
        <property name="suffix">
         <string> μs</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QDoubleSpinBox" name="surfaceLatest">
        <property name="suffix">
         <string> μs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="threshold">
        <property name="text">
         <string>触发阈值</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="surfaceOffset">
        <property name="suffix">
         <string> μs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelCustom">
        <property name="text">
         <string>表面跟踪偏移</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="surfaceThreshold">
        <property name="suffix">
         <string> %</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
